charts/m4d-crd/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: m4d-crd
+description: M4D CustomResourceDefinitions
+version: 0.0.0
+appVersion: 0.0.0
```

Comment:
appVersion is optional and we should probably omit it

Reply:
will omit